README.md

```diff
@@ -1,149 +1,114 @@
-     - REACT_APP_WEATHER_API_KEY
-        - this env var can be obtained from [open weather API](https://openweathermap.org/api). The API is used
-        to load current weather information in the application home page after logging in.
-  ### Start the application:
-  1. cd LiteFarm
-  2. `lerna bootstrap` to install dependencies
-  3. in separate terminals:
-      - `cd packages/webapp && npm start`
-      - `cd packages/api && npm start`
-      - (for hot reloading in api: npm install -g nodemon && nodemon --exec npm start)
-  4. webapp will be running on http://localhost:3000 and the api on http://localhost:5000
-  5. Since this is a mobile web application, webapp should be viewed in a mobile view in the browser
-
-# Testing:
-
-### Webapp:
-  Automated testing for the front-end is done using [cypress](https://www.cypress.io/).
-  To run automated tests in a chrome browser:
-  - Ensure that the webapp portal is running by doing `npm start` in the root directory of the webapp package
-  - `npm run cypress-ui` in the root directory of the webapp package
-
-  The test modules are stored in `packages/webapp/cypress/integration`.
-  Additional tests can be added by adding more testing modules to this directory.
-
-  Unit tests can also be run by:
-  - `npm run test`.
-
-  Unit tests reside in `src/tests`, and additional unit tests can be added in this directory.
-
-  We are currently using [ESLint](https://eslint.org/) to maintain code style in the app.
-  The linter can be run by `npm run lint`.
-
-### Api:
-  There is currently end-to-end testing being done for the LiteFarm API using the [chai.js](https://www.chaijs.com/)
-  and [jest](https://jestjs.io/) libraries.
-  The test files can be found in `packages/api/tests`.
-
-  All of the tests run real queries to the database using a jwt token obtained through Auth0.
-  Therefore, locally run tests can affect the state of the db. This is addressed by running a script which deletes
-  test data that is generated during the tests. However, this is not fool-proof, and
-  it may be necessary to clear the local db in some cases.
-
-  To run an end-to-end test, `npm run e2e` in the api directory.
-
-  To run a test file with a specific name:
-   - `npm start`
-   - `jest test_name`
-
-  It should be noted that any merges to develop and master branches in github will result in the code going through a CI pipeline, which
-  runs the end-to-end test. Therefore it is important to test locally first before creating any pull requests
-  to develop.
-
-  We are currently using [ESLint](https://eslint.org/) to maintain code style in the app. The linter can be run by `npm run lint`.
+- `packages/api`: the back-end API server
+- `packages/webapp`: the client-facing application
+
+## Preliminaries 
+
+1. Install [node.js](https://nodejs.org/en/download/package-manager/) if you do not already have it.
+3. Use the `git clone` command to clone this repository to your computer.
```

Comment:
Is this supposed to be `2.`?

Reply:
Yeah. Sorry, I use a WYSIWYG markdown editor that renders the numbers correctly no matter what the sequence. I must remember to proofread markdown in raw mode.